xcp/net/biosdevname.py

```diff
@@ -74,6 +74,10 @@ def all_devices_all_names():
                  "BIOS device" not in dinfo ):
                 continue
 
+            # threat USB devices a PCI
```

Comment:
DYM "treat" ?   "Threat" is something rather different.

Reply:
Fixed